assemble.bat

```diff
@@ -0,0 +1,9 @@
+set crate=bl602-hal
+
```

Comment:
You aren't deleting bin\*.a here, but are in the other scripts.
Let's keep the process consistent